ddpui/management/commands/migrate_github_to_ui4t.py

```
@@ -0,0 +1,337 @@
+        self.stdout.write(
+            self.style.SUCCESS(
+                f"  Updated tab preference for {updated_count}/{total_users} user(s)"
+            )
+        )
+
+    def _run_migration(
+        self,
+        org: Org,
+        orgdbt: OrgDbt,
+        warehouse: OrgWarehouse,
+        dry_run: bool = False,
```

Ruff (0.14.13)

Unused method argument: `dry_run`

(ARG002)